cornell-dti/office-hours
# Made minor changes to landing page

### Summary 


Made very minor fixes to the landing page.

I added the second curved ribbon right above the "for professors" header, I replaced the demo image with a new one, and I also added the logo in the footer. 
UPDATE: I forgot to add that I also removed the animation that fades in the login button and queue me in headers. This was bugged for some reason in the production version, so I got rid of it.





### Test Plan 
- Check that the ribbon, new demo image, and logo all look good for both desktop and mobile devices
- UPDATE: Check that the fade in animation for the login button and queue me in headers no longer works


### Notes 



### Breaking Changes 

None
